File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/enhetsregisteret/Virksomhet.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.enhetsregisteret;

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles.Orgnr;

public interface Virksomhet {
    public Orgnr getOrgnr();
    public String getNavn();
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/felles/Virksomhet.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller.felles

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller.felles.Næringskode
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller.felles.Orgnr

interface Virksomhet {
    val orgnr: Orgnr
    val navn: String
    val næringskode: Næringskode
}